client/blocks/comment-detail/style.scss

```diff
@@ -0,0 +1,322 @@
+		fill: $gray;
+		padding: 8px;
+		white-space: nowrap;
+		&:focus,
+		&:hover {
+			color: $blue-wordpress;
+			fill: $blue-wordpress;
+		}
+	}
+	span {
+		display: none;
+
+		@include breakpoint( ">960px" ) {
+			display: inline;
+		}
+	}
+}
+.comment-detail {
+	&.is-approved .comment-detail__action-approve .gridicon {
+		color: $alert-green;
+		fill: $alert-green;
+	}
+	&.is-liked .comment-detail__action-like .gridicon {
+		color: $alert-yellow;
+		fill: $alert-yellow;
+	}
+	&.is-spam .comment-detail__action-spam .gridicon {
+		color: $alert-red;
+		fill: $alert-red;
+	}
+	&.is-trash .comment-detail__action-trash .gridicon {
+		color: $gray-dark;
+		fill: $gray-dark;
+	}
+}
+
+
+.comment-detail__post {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: flex-start;
+	padding: 12px 16px;
+}
+
+.comment-detail__site-icon {
+	flex-shrink: 0;
+	width: 24px;
+	height: 24px;
+}
+.comment-detail__site-icon-image {
+	vertical-align: middle;
+}
+
+.comment-detail__post-info {
+	color: $gray;
+	overflow: hidden;
+	padding-left: 8px;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+
+	span {
+		color: $gray-text;
+		margin-right: 8px;
+	}
+
+	a {
+		color: $gray;
+		cursor: pointer;
+		fill: $gray;
+		&:focus,
+		&:hover {
+			color: $blue-wordpress;
+			fill: $blue-wordpress;
+		}
+	}
+}
+
+.comment-detail__comment {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	padding-left: 16px;
+}
+
+.comment-detail__comment-content {
+	border-left: 3px solid lighten( $gray, 30% );
+}
+
+.comment-detail__comment-body {
+	padding: 0 16px;
+}
+
+.comment-detail__comment-reply {
+	padding: 16px;
+
+	a {
+		color: $gray;
+		cursor: pointer;
+		fill: $gray;
+		&:focus,
+		&:hover {
+			color: $blue-wordpress;
+			fill: $blue-wordpress;
+		}
+	}
+}
+
+.comment-detail__author {
+	padding: 16px 0;
+
+	.comment-detail__author-info {
+		overflow: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+	}
+	.comment-detail__author-info-element {
+		overflow: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+	}
+}
+
+.comment-detail__author-preview {
+	align-items: center;
+	display: flex;
+	flex-flow: row nowrap;
+	padding: 0 16px;
+	transition: padding-bottom .3s linear;
+
+	.comment-detail__author-avatar {
+		height: 40px;
+		width: 40px;
+	}
+
+	.comment-detail__author-info {
+		line-height: 1.4;
+		padding: 0 8px;
+	}
+}
+.comment-detail.is-expanded .comment-detail__author-preview {
+	padding-bottom: 16px;
+}
+
+.comment-detail__author-more-info-toggle {
+	color: $gray;
+	cursor: pointer;
+	display: block;
+	fill: $gray;
+	height: 24px;
+	margin-left: auto;
+	&:focus,
+	&:hover {
+		color: $blue-wordpress;
+		fill: $blue-wordpress;
+	}
+}
+
+.comment-detail__author-more-info {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	color: $gray;
+	display: none;
+	transform: scaleY( 0 );
+	transform-origin: top;
+}
+.comment-detail__author.is-expanded .comment-detail__author-more-info {
```

Comment:
Unless things are particularly awkward to apply in bulk, I would recommend keeping the modifier as close to the actual element we're toggling since it keeps the rule short and the CSS specificity low.

It's optional here, but keep it in mind when we create very generic components, as it can be a pain to override rules when we get too specific.